src/Aspire.Hosting/ApplicationModel/InputAnnotationExtensions.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// Provides extension methods for <see cref="InputAnnotation"/>.
+/// </summary>
+internal static class InputAnnotationExtensions
+{
+    internal static T WithDefaultGeneratedPasswordAnnotation<T>(this T builder)
+        where T : IResourceBuilder<ContainerResource>
```

Comment:
Why only containers?

Reply:
I just needed Containers for now, and its internal. 

It'll change with my next PR when I need to add this to a BicepResource. I can change it now or later. Up to you.